js/script.js

```diff
@@ -217,9 +218,123 @@ function hello() {
           for (let i = 800; i < 3200; i += 100) {
             new_tag_map.set(i, 0);
           }
-          let tag_name =
+           tag_name =
             practice_each_topic[i].parentElement.firstChild.innerHTML;
           // //console.log(tag_name);
+         function generate_easy_question(arr, no) {
```

Comment:
I have generated exact same functions below(see line 346(old), same for generate_medium_question function and for hard also.